RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeeds_cfi.py

```diff
@@ -4,7 +4,94 @@
 # module to produce pixel seeds for electrons from super clusters
 # Author:  Ursula Berthon, Claude Charlot
 #
-from RecoEgamma.EgammaElectronProducers.ecalDrivenElectronSeedsParameters_cff import *
+
+from RecoTracker.IterativeTracking.ElectronSeeds_cff import newCombinedSeeds as _newCombinedSeeds
+
+ecalDrivenElectronSeedsParameters = cms.PSet(
```

Comment:
this PSet was apparently defined outside in view of possible reuse elsewhere.
If there is no need to reuse, it seems more practical to simply inline this in the ecalDrivenElectronSeeds definition